### PR TITLE
Add mumble tokens

### LINKED ISSUE #63: Token (channel password) is not working
Hi,
I've just setup a murmur + mumble-web installation. Everything works great, apart from one thing:

When I create a new channel (through Mumble) and assign it a password (= token), it works as it should in Mumble: Users have to add the password as an "Access token" to be able to join the channel.
However, in mumble-web even if I connect to the server with the correct "token" in the token field of the connect dialog, I cannot join the channel. The mumble-web UI just does nothing upon clicking Join, and the murmur logs state "testuser is not allowed to Enter in Testchannel" (testuser and Testchannel are the username / channel name in question).

Is this a known bug? Any recommendations on how I can further debug this?

### PR DESCRIPTION
**Pull request by encrypt94**
This adds mumble tokens for channel authentication (should close #63 )
A demo is available here: https://mumble.gattini.ninja/?tokens=antani,test&channelName=Test

About tokens in query, bot **token** and **tokens** are handled.
Tokens are comma separated  (like: ?tokens=token1,token2,token3) but i see that comma is an allowed character in mumble tokens so this will not work with tokens containing this character. 
I'm open to improve that, advices are welcome.

Also a quick edit on channelName.
Previously was mandatory to have channelName starting with '/' now if it's not present it's added automatically.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -894,8 +915,11 @@ window.onload = function () {
   } else {
     useJoinDialog = false
   }
+  if (queryParams.tokens) {
+    ui.connectDialog.tokens(queryParams.tokens.split(","))
+  }
   if (queryParams.token) {
-    ui.connectDialog.token(queryParams.token)
+    ui.connectDialog.tokens(queryParams.token.split(","))
```
Comment (Johni0702) [inline app/index.js]: Looks like if the same query parameter is given multiple times, the `url` lib (at least the nodejs version) will return an array instead:
```
> url.parse('https://a/?t=1&t=2', true).query.t
[ '1', '2' ]
```
We can probably use that to solve the token-which-contains-comma problem by not splitting `queryParams.token` but instead just adding it raw if it's a string (e.g. `?token=te,st`) or adding each entry raw if it's an array (e.g. `?token=te,st&token=tes,t`).

File: app/index.js [hunk 2]
```diff
@@ -362,9 +377,15 @@ class GlobalBindings {
         window.matrixWidget.setAlwaysOnScreen(true)
 
         // Register all channels, recursively
+	  
+	if(channelName.indexOf("/") != 0) {
+	  channelName = "/"+channelName;
+	}
         const registerChannel = (channel, channelPath) => {
           this._newChannel(channel)
+ 	  console.log(channelPath)
```
Comment (Johni0702) [inline app/index.js]: This looks like a debug message. If it is, please remove it, otherwise indent with spaces.

File: app/index.html [hunk 3]
```diff
@@ -41,9 +41,22 @@
...
+                  </td>
+                </tr>
+                <tr data-bind="if: $root.config.connectDialog.token">
+                      <td></td>
+		      <td>
+                        <button class="dialog-submit" type="button" data-bind="enable: selectedTokens().length > 0, click: removeSelectedTokens()">Remove</button>
+                        <button class="dialog-submit" type="button" data-bind="enable: tokenToAdd().length > 0, click: addToken()">Add</button>
+  		      </td>
                 </tr>
+		<tr data-bind="if: $root.config.connectDialog.token">
```
Comment (Johni0702) [inline app/index.html]: Might want to hide the token list while there are no token in it to reduce clutter in the connect dialog.

File: app/index.js (hunk 1 above)
Comment (encrypt94) [reply app/index.js]: Nice! works perfectly!

**Comment by encrypt94** [comment]
Everything done! Also demo instance updated.
Thanks for your feedback!

**Pull request merged**